src/IO/H5/VolumeData.hpp

```diff
@@ -210,15 +233,25 @@ class VolumeData : public h5::Object {
   std::vector<std::vector<Spectral::Quadrature>> get_quadratures(
       size_t observation_id) const;
 
-  /// Get the serialized domain in the subfile at this observation ID, or
-  /// `std::nullopt` if no domain was written.
-  std::optional<std::vector<char>> get_domain(size_t observation_id) const;
+  /*!
+   * \brief Get the serialized domain if it was written.
+   */
+  std::optional<std::vector<char>> get_domain(
+      std::optional<size_t> observation_id = std::nullopt) const;
 
-  /// Get the serialized functions of time in the subfile at this observation
-  /// ID, or `std::nullopt` if none were written.
+  /*!
+   * \brief Get the observation-specific serialized functions of time at an \p
+   * observation_id if they were written.
+   */
   std::optional<std::vector<char>> get_functions_of_time(
       size_t observation_id) const;
```

Comment:
Do we still write them at observation times? I thought this would now always be the global ones?